HappyAnding/HappyAnding/Views/ExploreShortcutViews/ExploreCategoryView.swift

```diff
@@ -7,14 +7,130 @@
+    
+    let category: String
+    let categoryDescriptions = [
+        "교육":"교육 카테고리에 대한 설명입니다.",
+        "금융":"금융 카테고리에 대한 설명입니다.",
+        "비즈니스":"비즈니스 카테고리에 대한 설명입니다.",
+        "건강 및 피트니스":"건강 및 피트니스 카테고리에 대한 설명입니다.",
+        "라이프스타일":"라이프스타일 카테고리에 대한 설명입니다.",
+        "날씨":"날씨 카테고리에 대한 설명입니다.",
+        "사진 및 비디오":"사진 및 비디오 카테고리에 대한 설명입니다.",
+        "꾸미기":"꾸미기 카테고리에 대한 설명입니다.",
+        "유틸리티":"유틸리티 카테고리에 대한 설명입니다.",
+        "소셜 네트워킹":"소셜 네트워킹 카테고리에 대한 설명입니다.",
+        "엔터테인먼트":"엔터테인먼트 카테고리에 대한 설명입니다.",
+        "여행":"여행 카테고리에 대한 설명입니다."
+    ]
+    
+    var body: some View {
+        VStack {
+            List {
+                Text(categoryDescriptions[category] ?? "")
+                    .foregroundColor(.Gray5)
+                    .Body2()
+                    .padding(16)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .background(
+                        Rectangle()
+                            .foregroundColor(Color.Gray1)
+                            .cornerRadius(12)
+                    )
+                    .listRowInsets(
+                        EdgeInsets(
+                            top: 20,
+                            leading: 16,
+                            bottom: 0,
+                            trailing: 16
+                        )
+                    )
+                    .listRowSeparator(.hidden)
+                    .listRowBackground(Color.Background)
+                Section() {
+                    ListHeader(title: "다운로드 순위")
+                        .padding(.top, 20)
+                        .listRowBackground(Color.Background)
+                    ForEach(rankingShortcuts.indices, id: \.self) { index in
+                        if index < 3 {
+                            ShortcutCell(
+                                color: rankingShortcuts[index].color,
+                                sfSymbol: rankingShortcuts[index].sfSymbol,
+                                name: rankingShortcuts[index].name,
+                                description: rankingShortcuts[index].description,
+                                numberOfDownload: rankingShortcuts[index].numberOfDownload,
+                                downloadLink: rankingShortcuts[index].downloadLink
+                            )
+                        }
+                    }
+                    .listRowInsets(EdgeInsets())
+                    .listRowSeparator(.hidden)
+                }
+                Section() {
+                    ListHeader(title: "사랑받는 단축어")
+                        .padding(.top, 20)
+                        .listRowBackground(Color.Background)
+                    ForEach(lovedShortcuts.indices, id: \.self) { index in
+                        if index < 3 {
+                            ShortcutCell(
+                                color: lovedShortcuts[index].color,
+                                sfSymbol: lovedShortcuts[index].sfSymbol,
```

Comment:
lovedShortcuts[index] 같이 index로 데이터를 다루면 할당되지않은 메모리에 접근할 수 있어서 지양하는 것이 좋다고 하더라구요!

ForEach문 코드가 조금더 길어지긴 하는데


```suggestion
ForEach(Array(lovedShortcuts.enumerated()), id:\.offset) { index, shortcut in
                                ...
                                sfSymbol: shortcut.sfSymbol,
                                ...
```
이런 방법도 있을 것 같습니다!